libs/common-types/src/milestone_origin.rs

```diff
@@ -47,7 +47,7 @@ pub enum TreasuryOriginError {
     Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, MaxEncodedLen,
 )]
 pub enum FundingType {
-    Proposal,
+    Project,
```

Comment:
In retrospect, this makes more sense to be called Crowdfund since all 3 funding types are eventually projects